diff -Nur szip-2.1-orig/examples/Makefile.am szip-2.1/examples/Makefile.am
--- szip-2.1-orig/examples/Makefile.am	2004-11-17 21:54:43.000000000 +0100
+++ szip-2.1/examples/Makefile.am	2009-09-10 10:29:32.000000000 +0200
@@ -11,7 +11,7 @@
 noinst_PROGRAMS = burst_szip example
 
 burst_szip_SOURCES = burst_szip.c
-burst_szip_LDADD =  ../test/libmcgill.la ../src/libsz.la
+burst_szip_LDADD =  ../test/libmcgill.la ../src/libsz.la -lm
 burst_szip_DEPENDENCIES = ../test/libmcgill.la ../src/libsz.la
 
 
